authui/src/customSelect.ts

```diff
@@ -50,6 +50,7 @@ export class CustomSelectController extends Controller {
 
   keyword = "";
   value?: string;
+  focusedValue?: string;
```

Comment:
Do not introduce redundant instance properties when the source of truth is the attributes of some elements in our control.